docs/source/markdown/podman-images.1.md

```diff
@@ -25,27 +25,32 @@ Show image digests
 
 #### **--filter**=*filter*, **-f**
 
-Filter output based on conditions provided
+Provide filter values.
 
-  Filters:
+The *filters* argument format is of `key=value`. If there is more than one *filter*, then pass multiple OPTIONS: **--filter** *foo=bar* **--filter** *bif=baz*.
 
-  **since=IMAGE**
-    Filter on images created after the given IMAGE (name or tag).
+Supported filters:
 
-  **before=IMAGE**
-    Filter on images created before the given IMAGE (name or tag).
+| Filter             | Description                                                                                   |
+| :----------------: | --------------------------------------------------------------------------------------------- |
+| *before*           | Filter by images created before the given IMAGE (name or tag).                                |
+| *dangling*         | Filter by dangling (unused) images.                                                           |
+| *label*            | Filter by images with (or without, in the case of label!=[...] is used) the specified labels. |
+| *readonly*         | Filter by read-only or read/write images.                                                     |
+| *reference*        | Filter by image name.                                                                         |
+| *since*            | Filter by images created after the given IMAGE (name or tag).                                 |
 
-  **dangling
-    Show dangling images. Dangling images are a file system layer that was used in a previous build of an image and is no longer referenced by any image. They are denoted with the `<none>` tag, consume disk space and serve no active purpose.
+The `before` *filter* accepts formats: `<image-name>[:<tag>]`, `<image id>` or `<image@digest>`.
 
-  **label**
-    Filter by images labels key and/or value.
+The `dangling` *filter* shows images that are taking up disk space and serve no purpose. Dangling image is a file system layer that was used in a previous build of an image and is no longer referenced by any image. They are denoted with the `<none>` tag, consume disk space and serve no active purpose.
```

Comment:
```suggestion
The `dangling` *filter* shows images that are taking up disk space and serve no purpose. A dangling image is a file system layer used in a previous build of an image and is no longer referenced by any image. They are denoted with the `<none>` tag, consume disk space, and serve no active purpose.
```

Reply:
Too late for this. This is a backport, any changes should be made first in main, then backported in a future RC.